deeppavlov/models/morpho_tagger/network.py

```diff
@@ -321,11 +324,74 @@ def load(self, infile) -> None:
 
 
 @register("morpho_tagger")
-class MorphoTagger(KerasWrapper):
+class MorphoTagger(KerasModel):
     """
     A wrapper over :class:`CharacterTagger`.
     It is inherited from :class:`~deeppavlov.core.keras_model.KerasWrapper`.
     It accepts initialization parameters of :class:`CharacterTagger`
     """
-    def __init__(self, *args, **kwargs) -> None:
-        super().__init__(CharacterTagger, *args, **kwargs)
\ No newline at end of file
+    def __init__(self, save_path: Optional[str] = None,
+                 load_path: Optional[str] = None, mode: str = None,
+                 **kwargs) -> None:
+        # Calls parent constructor. Results in creation of save_folder if it doesn't exist
+        super().__init__(save_path=save_path, load_path=load_path, mode=mode)
+
+        # Dicts are mutable! To prevent changes in config dict outside this class
+        # we use deepcopy
+        opt = deepcopy(kwargs)
+
+        # Finds all input parameters of the network __init__ to pass them into network later
+        network_parameter_names = list(inspect.signature(CharacterTagger.__init__).parameters)
+        # Fills all provided parameters from opt (opt is a dictionary formed from the model
+        # json config file, except the "name" field)
+        network_parameters = {par: opt[par] for par in network_parameter_names if par in opt}
+        self._net = CharacterTagger(**network_parameters)
```

Comment:
Can we rename `CharacterTagger` into `MorphoTagger` and move all the wrapper logic there? It should become a lot more transparent that way.